packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -44,6 +44,7 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
     // eth_requestAccounts
     handshake: async (_: RequestArguments): Promise<AddressString[]> => {
       if (this.connected) {
+        this.emit('connect', { chainId: prepend0x(this.chain.id.toString(16)) });
```

Comment:
turns out we still need this, when dapp reloads and they dont rememebr logged in state, we'd need to alert them